small_gicp_node:
  ros_parameter:
    pcd_path: "src/"
    num_threads : 4
    num_neighbors : 20
    leaf_size: 0.25
    max_dist_sq: 1.0
    